src/styles/_catalog.scss

```
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: all 300ms;
```

```suggestion
    transition: color 300ms, transform 300ms;
```
вказувати all - не найкраща ідея. Потрібно точно вказувати, до якої властивості буде застосовано transition